### ISSUE TITLE
Unable to create new field

### ISSUE THREAD

**Issue by mroswell** [mbloch/mapshaper#206]
I don't get a new field upon running this:

`$ mapshaper cb_2014_us_county_20m.json -verbose -each 'CTY_FIPS=STATEFP+COUNTYFP'`
- 142ms -i
- 14ms -each

**Comment by mbloch** [comment]
The command worked for me when I tested as follows: First I converted the  cb_2014_us_county_20m.shp Shapefile downloaded from data.gov to GeoJSON using `mapshaper cb_2014_us_county_20m.shp -o format=geojson`, then I ran the command and got the output shown below. The `-info` command shows that CTY_FIPS was created.

If you are certain that the CTY_FIPS command did not work for you, could you give me some more information?
* What operating system are you using?
* What version of mapshaper do you have? (mapshaper -v)
* What version of Node do you have? (node -v)
* Does the command work for you if you run it in the web app at mapshaper.org?

Thanks...

```
$ mapshaper cb_2014_us_county_20m.json -verbose -each 'CTY_FIPS=STATEFP+COUNTYFP' -info
- 142ms -i
- 11ms -each
[info] 
Layer 1
Layer name: cb_2014_us_county_20m
Records: 3,220
Geometry
  Type: polygon
  Bounds: -179.174265 17.913769 179.773922 71.352561
  Proj.4: +proj=longlat +datum=WGS84
Attribute data
  Field     First value
  AFFGEOID  '0500000US16017'
  ALAND     4492522387
  AWATER     478947473
  COUNTYFP  '017'
  COUNTYNS  '00395158'
  CTY_FIPS  '16017'
  GEOID     '16017'
  LSAD      '06'
  NAME      'Bonner'
  STATEFP   '16'
- 33ms -info
```

**Comment by mroswell** [comment]
MacOS Sierra
10.12.6 (16G29)

$ npm -v
4.2.0

$ node -v
v7.10.0

Why, yes it does work when I run it in the console of mapshaper.org. Good to know about that option!

I reran the steps locally and has these results
- no change on 2014 
- once saw output on a 2015 version (turned out I was running the -each and the -info at the same time, that time) 
- when I went back and looked at -info for the file (without running -each again) the CTY_FIPS wasn't there, even in the 2015 version

Maybe it has something to do with the file permissions on my system.

The -o commandscreated 
-rw-r--r--   1 mydir  staff  2384089 Aug  8 07:21 cb_2015_us_county_20m.json

(And the same permissions for 2014 output)

**Comment by mbloch** [comment]
Aha, here's what might be happening. Originally, the command line program would always save a file - as though you had added the `-o` command every time. Looking at the CHANGELOG, I see that I stopped automatically creating output for version 0.2.18. Now you have to add `-o` (followed by any output options that you want).

Could it be that you've been running mapshaper without `-o` some of the time?

**Comment by mroswell** [comment]
That's it! This (and probably other locations in the documentation as well) should be updated accordingly.
https://github.com/mbloch/mapshaper/wiki/Command-Reference#-each

(Though I wouldn't mind if the -o was changed back to appearing by default, unless there's some good reason not to.)

**Comment by mbloch** [comment]
I think I removed automatic saving because I often use mapshaper to get information about a file, and don't want to save output. I'm open to adding a warning message if mapshaper is run with an editing command but no `-o` command.

**Comment by mroswell** [comment]
Whether warning or no warning, the docs need to be updated accordingly. My preference would be output by default, with a flag to suppress it. (And maybe an option, so that even if someone has written a command to default to the --suppress flag, if -o is the last flag, it wins)

(And foolish me, the original dataset had a field called "GEOID" that already had these two FIPS codes concatenated. But glad to have surfaced the content in this thread, particularly learning of the console option at mapshaper.org, and recommending that docs match the output behavior.)

**Comment by mbloch** [comment]
You're suggesting that the examples in the wiki should have an `-o` command appended to them, is that right? It's true, the examples are incomplete without `-o` commands. I'm happy to update the docs.

I don't want to make an implicit `-o` the default, for the following reason: by making `-o` implicit, it is very easy to overwrite a source file by accident, which could cause users to lose their valuable data.

**Comment by mbloch** [comment]
I updated the command reference to add `-o` commands where indicated. Also released v0.4.43, which requires the `-o force` option to overwrite input files.
